lib/utils.sh

```diff
@@ -617,58 +622,66 @@ asdf_run_hook() {
   fi
 }
 
-with_shim_executable() {
-  get_shim_versions() {
-    shim_plugin_versions "${shim_name}"
-    shim_plugin_versions "${shim_name}" | cut -d' ' -f 1 | awk '{print$1" system"}'
-  }
```

Comment:
Move these functions outside of the `with_shim_executable` function to make things simpler to read.